formula/src/test/java/com/instacart/formula/test/CoroutineTestableRuntime.kt

```diff
@@ -70,20 +76,30 @@ object CoroutinesTestableRuntime : TestableRuntime {
 }
 
 private class FlowRelay : Relay {
-    private val sharedFlow = MutableSharedFlow<Unit>(0, 1)
+    private val sharedFlow = MutableSharedFlow<Unit>(0, 0)
 
     override fun action(): Action<Unit> = FlowAction.fromFlow { sharedFlow }
 
+    @OptIn(DelicateCoroutinesApi::class)
     override fun triggerEvent() {
-        sharedFlow.tryEmit(Unit)
+        GlobalScope.launch(start = CoroutineStart.UNDISPATCHED) {
```

Comment:
Combining `UNDISPATCHED` with `Unconfined` to ensure that we emit the value to a relay like flow immediately.